Automattic/wp-calypso
# E2E: Add e2eflowtestingi18nuser into config.

#### Changes proposed in this Pull Request

This PR adds a new user, e2eflowtestingi18nuser to the encrypted configuration file.

#### Testing instructions

1. pull the branch
2. go to `test/e2e`
3. run `npm run decryptconfig`
4. open the decrypted file
5. verify `e2eflowtestingi18nuser` is present.

Closes https://github.com/Automattic/wp-calypso/issues/57209
